### PR TITLE
More robust cloud_sql_proxy usage

### PR DESCRIPTION
**Pull request by dmohs**
Causes proxy failures to happen early to avoid "why the hell isn't my database working" moments.

Tested by calling the `/v1/me` endpoint which didn't return a 500.

### CODE REVIEW

File: api/libproject/cloudsqlproxycontext.rb [hunk 1]
```diff
@@ -5,23 +5,25 @@
...
-        sleep 1 # TODO(dmohs): Detect running better.
-        yield
-      ensure
-        Process.kill "HUP", @ps
-        Process.wait
-      end
+    @ps = fork do
+      exec *%W{
+        cloud_sql_proxy
+          -instances #{@project}:us-central1:workbenchmaindb=tcp:0.0.0.0:3307
```
Comment (rushtong) [inline api/libproject/cloudsqlproxycontext.rb]: Please excuse my drive-by comment. I know `us-central1` is the current region, but will this always be true? Should we pull that into a config somewhere?

Comment (dmohs) [reply api/libproject/cloudsqlproxycontext.rb]: Good question. My philosophy is that we should put it into a config as soon as we have two and no sooner.

**Comment by calbach** [comment]
I don't have context on this - is it still in review? Should it be dropped at this point?

**Comment by rushtong** [comment]
@calbach - I think this is stale. Unless someone wants to advocate for it, and make a story for it, I suggest we close it.

**Comment by dmohs** [comment]
Updated so you can take it if you want it.

**Comment by calbach** [comment]
AFAIK no one has hit this issue recently, so I'll close this PR for now. Will leave the branch around in case we determine we need to resurrect it.
